### ISSUE TITLE
BibTex and ReferenceManager export - property of "unpublished" changes "item type" to "unpublished" ?

### ISSUE THREAD

**Issue by photomedia** [eprints/eprints3.4#123]
This line sets the "type" in the BibTex export to "unpub" for any type (thesis, article, etc.) if it finds the "unpublished" value in "ispublished" field:
 "https://github.com/eprints/eprints3.4/blob/d92e3175b75aaa66513613aedfa95e842e5cfb66/flavours/pub_lib/plugins/EPrints/Plugin/Export/BibTeX.pm#L155

I don't think that makes sense.  A thesis might be "unpublished" but it is nevertheless still a "thesis" that is available in a repository.  Furthermore, this breaks the import into Zotero, as there is no "unpublished" as an Item Type there.

The same thing happens with Reference Manager export, on this line:

Both ReferenceManager and BibTex would otherwise work for Zotero export if it wasn't for this unexpected issue with "unpublished":

https://github.com/eprints/eprints3.4/blob/d92e3175b75aaa66513613aedfa95e842e5cfb66/flavours/pub_lib/plugins/EPrints/Plugin/Export/RIS.pm#L358

Are we sure that this use of "unpublished" as an item type is correct?

**Comment by drn05r** [comment]
I think I agree with your point. I think the best solution is changing the conditional statement that will only change the type (to unpublished or UNPB) if it has not already been changed by the fact that it is specific type.   Although reading the BibTeX, this would technically be wrong.  However, I guess this is where theory and the real world clash.  Two things that could help:

1. Add a note field to indictate it is unpublished whilst keeping the original type (i.e. article, thesis, etc.).
2. Make this feature of not setting as unpublished confiigurable but enabled by default, i.e. future releases by default won't set unpublished if a specific type has already been set.

**Comment by photomedia** [comment]
Hi @drn05r!  Thank you so much for making this change.  I just tested this with a default 3.4.3, and I see that the "unpublished" overrides the set item type, even though the option is set.  I think that contradicts the intention of NOT overriding by default?
"Make this feature of not setting as unpublished confiigurable but enabled by default, i.e. future releases by default won't set unpublished if a specific type has already been set."?

This code: 
`$publication_status_type_override = 1 unless EPrints::Utils::is_set( $publication_status_type_override );`
So in other words, set it to override (ie., unpublished) by default, unless it is already set to 1 in config (which it is, by default).
So to get it to NOT override, I would have to set the config option to 0, right?

**Comment by drn05r** [comment]
Yes.  If there is no configuration option `$c->{export}->{publication_status_type_override}` then the variable `$publication_status_type_override` will be set to 1, (i.e. true).  The only way for `$publication_status_type_override`  to be false is if you explicitly set `$c->{export}->{publication_status_type_override} = 0;`
